app/commands/markdown/preprocess.rb

```diff
@@ -1,9 +1,17 @@
 class Markdown::Preprocess
   include Mandate
 
-  initialize_with :text
+  def initialize(text, strip_h1: true) # rubocop:disable Naming/VariableNumber
```

Comment:
Rubocop doesn't like the `h1` suffix. I've disabled it via comments for now. Let me know if that makes sense.

Reply:
Replied to this above.